src/components/alert/alert.tsx

```diff
@@ -166,7 +181,11 @@ export class Alert implements OpenCloseComponent {
         <calcite-icon icon="x" scale={this.scale === "l" ? "m" : "s"} />
       </button>
     );
-    const queueText = `+${this.queueLength > 2 ? this.queueLength - 1 : 1}`;
+    const queueNumber = this.queueLength > 2 ? this.queueLength - 1 : 1;
+    const queueText = "+".concat(
```

Comment:
Is there a way to integrate the `+` into the formatted string? Concatenating isn't recommended and won't be correct for all locales.